ansible/elasticsearch.yml

```
@@ -16,6 +16,19 @@
     es_xpack_features: [] # disable features
     es_instance_name: "{{ ansible_hostname }}"
 
+    # Note: while elasticsearch stores some user provided data, it normally does
```

```suggestion
    # Protect against potential information leak.  
    # For details see https://docs.wire.com/security-responses/log4shell.html
    #
    # Note: while elasticsearch stores some user provided data, it normally does
```

If you want to be extra nit-picky, you can move the entire comment block closer to line 30.

---

thanks!